cpp/transfer-learning/classify.cpp

```diff
@@ -0,0 +1,82 @@
+//
+//  classify.cpp
+//  transfer-learning
+//
+//  Created by Kushashwa Ravi Shrimali on 15/08/19.
+//  Copyright © 2019 Kushashwa Ravi Shrimali. All rights reserved.
+//
```

Comment:
Do you think it would be possible to remove the copyright notice here? I looked at other examples in this repo and they don't seem to have copyright notices. Also having the notice might discourage people from using / modifying the code.

Reply:
Hi, sorry I missed this. This was automatically put by the IDE I used and I forgot to remove it. Thanks! :)